components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -112,15 +112,19 @@
               <chef-form-field class="attribute-field">
                 <label>
                   <span class="label">Value</span>
-                  <chef-input ngDefaultControl
-                    formControlName="values"
-                    [value]="condition.controls.values.value"
-                    data-cy="rule-value"
-                  ></chef-input>
+                  <input
+                  chefInput
+                  formControlName="values"
+                  [value]="condition.controls.values.value"
+                  data-cy="rule-value"
+                  />
                 </label>
-                <chef-error
-                  *ngIf="(ruleForm.get('conditions').controls[i].get('values').hasError('required') || ruleForm.get('conditions').controls[i].get('values').hasError('pattern')) && ruleForm.get('conditions').controls[i].get('values').dirty"
-                >Value is required.</chef-error>
+                <div>
+                  <chef-error
+                      *ngIf="( ruleForm.get('conditions').controls[i].get('values').hasError('required') && ruleForm.get('conditions').controls[i].get('values').touched )
+                              || ( ruleForm.get('conditions').controls[i].get('values').hasError('pattern') && ruleForm.get('conditions').controls[i].get('values').dirty )"
+                    >Value is required.</chef-error>
+                </div>
```

Comment:
I changes these so that we're checking two different setups now
1.  If the input doesn't have a value AND it has been touched - we need to show the error
OR
2.  If the input has a value (is dirty) AND that value does not meet our pattern criteria, we are showing an error